gadopt/approximations.py

```diff
@@ -166,13 +182,16 @@ def rhocp(self):
         return 1
 
     def kappa(self):
-        return self.thermal_diffusivity
+        return self.kappa_ref
```

Comment:
What is this change for?

Reply:
I am just reverting something I did in the documentation PR and do not like anymore. `thermal_diffusivity` just does not fit with how other attributes are named. Before, there was `_kappa`, which does not work either because it implies that the attribute is private, which I believe is not intended. I decided to put `kappa_ref` instead. It could be `kappa`, but in that case, the method's name needs to change.